plugins/@grouparoo/app-templates/src/source/table/profileProperties.ts

```diff
@@ -55,6 +56,14 @@ export const getProfileProperties: GetProfilePropertiesMethod = ({
         if (!primaryKeysHash[k]) primaryKeysHash[k] = [];
         primaryKeysHash[k].push(profiles[i].id);
       }
+      if (
+        aggregationMethod === AggregationMethod.Sum ||
+        aggregationMethod === AggregationMethod.Count
+      ) {
+        //default all entries to 0 to start... will be replaced later if there is a response
+        responsesById[profiles[i].id] = [];
+        responsesById[profiles[i].id].push(0);
+      }
     }
```

Comment:
Helpful comment! 

No change needed, but this could be slightly written as `responsesById[profiles[i].id] = [0]`, skipping the `push`